config/locales/service_information_banner.yml

```
@@ -1,17 +1,17 @@
 en:
   service_information_banner:
     provider:
-      sandbox_header: The sandbox will be unavailable on Tuesday 29 October from 7am to 8am
+      sandbox_header: The sandbox will be unavailable on Thursday 11 September from 7am for approximatively 3 hours
       sandbox_body: This will affect both the web service and the API. You may lose work if you are using the sandbox when it becomes unavailable.
-      header: The Manage service will be unavailable on Tuesday 29 October from 7am to 8am
+      header: The Manage service will be unavailable on Thursday 18 September from 7am for approximatively 3 hours
       body: This will affect both the web service and the API. You may lose work if you are using Manage when it becomes unavailable.
     candidate:
-      sandbox_header: The sandbox will be unavailable on Tuesday 29 October from 7am to 8am
+      sandbox_header: The sandbox will be unavailable on Thursday 11 September from 7am for approximatively 3 hours
```

```suggestion
      sandbox_header: The sandbox will be unavailable on Thursday 11 September from 7am for approximately 3 hours
```

approximatively is a word, but not commonly used. I think approximately, is probably what was intended. (on lines 6, 11, 14, and 16)